static/app/api.tsx

```diff
@@ -581,3 +585,60 @@ export class Client {
     );
   }
 }
+
+function generateOrganizationBaseUrl(organizationUrl: string) {
+  return `${organizationUrl}${DEFAULT_BASE_URL}`;
+}
+
+type APIRouteType = 'organization-events';
```

Comment:
Will this become a union of all the endpoint names?

Reply:
@markstory yep. This will be a union type.